app/src/main/kotlin/com/getstrm/pace/processing_platforms/ProcessingPlatformViewGenerator.kt

```diff
@@ -194,7 +173,7 @@ abstract class ProcessingPlatformViewGenerator(
         if (fieldTransform.transformsList.size == 1) {
             // If there is only one transform it should be the only option
             val (_, queryPart) = toCase(fieldTransform.transformsList.last(), field)
-            return queryPart.`as`(field.fullName())
+            return queryPart.`as`(fieldAlias(field.fullName(), fieldTransform.transformsList.last()))
```

Comment:
why the last one and not the first, or one in the middle?

Reply:
see the if statement. There is only one.